client/components/domains/registrant-extra-info/style.scss

```diff
@@ -31,6 +20,12 @@
 			margin: 0;
 		}
 	}
+
+	.form-label .form-label__optional {
+		font-weight: 200;
```

Comment:
To my eye at least, you could remove this font-weight declaration. The weight appears too undernourished compared with the page's overall typographical styles.

<img width="556" alt="screen shot 2018-02-28 at 4 24 01 pm" src="https://user-images.githubusercontent.com/6458278/36824033-7085ef46-1d54-11e8-9d0e-378b259e0e8b.png">

<img width="584" alt="screen shot 2018-02-28 at 4 24 09 pm" src="https://user-images.githubusercontent.com/6458278/36824036-733ea304-1d54-11e8-81b2-57781e6a4370.png">